.travis.yml

```diff
@@ -13,7 +13,6 @@ env:
 
 install: skip
 before_script:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
Also removed `yarn` update because:

1) after these years the bundled `yarn` version in CI is reasonably new.
2) `yarnpkg.com` could fail, i.e. https://travis-ci.com/github/JLHwung/babel/jobs/321733324